### DISCUSSION TITLE
Allow Date Format Selection

### DISCUSSION THREAD

**Discussion by bradmcdowell** [mealie-recipes/mealie#4571] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

The date format is currently set to the American MM/DD/YYYY format.

In Australia, we use the DD/MM/YYYY format. I'm sure there are many other date formats that other countries prefer.

### Please provide a concise description of the feature that would resolve your issue.

Can we either pick up the date format based on the browser settings?

or

Have a feature where administrators can set the date format for the application.

### Please consider and list out some caveats or tradeoffs made in your design decision

None that I can think of.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
I'm pretty sure date format is tied to language; have you tried switching to British English?

(I know Australia and Britain are two completely different places, only suggesting it for the date format)

**Reply by bradmcdowell** [reply]
Hi Michael,

You are right. I set it to British English and the date format displays corectly.

The next question is, how can we set the default language for all users?

**Reply by michael-genson** [reply]
Currently everyone needs to set their language themselves AFAIK

**Reply by tecosaur** [reply]
FWIW, I know I'd appreciate a default language option: currently every friend and family member needs to be told to individually set this, which is not ideal.
